aya/src/obj/btf/btf.rs

```diff
@@ -399,6 +399,18 @@ impl Btf {
                     fixed_ty.name_offset = 0;
                     types.types[i] = BtfType::Ptr(fixed_ty)
                 }
+                // Fixup STRUCT for Rust
```

Comment:
We should probably create issues for those fixups, to not lose track of them and think of proper LLVM/rustc fixes. I'm curious - what are the examples of such names?